src/features/docks/selectors.ts

```diff
@@ -10,7 +11,7 @@ import { selectOrdered } from '~/utils/collections';
  * on the UI.
  */
 export const getDocksInOrder = createSelector(
-  (state) => state.docks,
+  (state: RootState) => state.docks,
```

Comment:
Similarly to beacons, I would introduce a `getDocksAsCollection` selector and explicitly annotate all other selectors as well.